Earthfile

```diff
@@ -19,7 +21,6 @@ docs:
     WORKDIR ./docs
 
     RUN npm install postcss-cli
-    RUN npm run prepare
```

Comment:
oh shit, this is gonna improve the building of the arm64 packages by a lot!

Reply:
maybe I should test that and make sure I didn't break anything there... sounds too good to be  true :P